clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -15,14 +15,16 @@
 
 package tachyon.client.file.options;
 
+import com.google.common.base.Optional;
+
 import tachyon.client.ClientContext;
 import tachyon.conf.TachyonConf;
+import tachyon.thrift.SetStateTOptions;
 
 public class SetStateOptions {
   public static class Builder {
-    private Boolean mPinned;
-    // TODO(cc) Have an option to change TTL, would be used in a shell command like "setTTL", should
-    // also update logic of TTLBucketList when ttl value of an existing file is updated.
+    private Optional<Boolean> mPinned;
```

Comment:
What do you mean unsafe? Are you worried that the callers of the getters might not expect the return value to potentially be null? If so, you can call the getters `getPinnedOrNull` and `getTTLOrNull`.